tox.ini

```diff
@@ -11,7 +11,7 @@ deps =
     sphinx53: sphinx==5.3.*
     sphinx62: sphinx==6.2.*
     sphinx70: sphinx==7.0.*
-    sphinx71: sphinx==7.1.*
```

Comment:
Isn't this considered major release? I would consider keeping this for now.

Reply:
Given all the breakage from yesterday, I guess, LoL. Can I drop "7.0" then? I don't want to keep a super long list of old Sphinx releases.

Reply:
I would say wait? It's released back in April, and maybe there are still some <7.1 pins around. I don't expect much failures, but then we also don't run the test suite that often that it should be that big of a deal to cover all cases.

Dropping down to a single version for old major numbers sounds good though.

Reply:
Do you have a preference between 5.3 and 6.2?

Reply:
> Wouldn't it make sense to pick up the bug fixes too?

No, I think that would be the same logic of supporting only the latest and greatest version, as that one has the most features and fixes.

My logic here is that the config files state a minimum required version, our task is to make sure that minimum is in fact the minimum and not just aspiration, therefore that minimum should be the one tested.

Reply:
I think I addressed your comments but I am not sure. Please re-review. Thanks!

Reply:
Oh, I mean keep one of each of the major versions (e.g. don't drop 5.x.y), but one should be enough, no need for multiple between 5.x and 6.y